libafl_qemu/src/aarch64.rs

```diff
@@ -0,0 +1,46 @@
+use num_enum::{IntoPrimitive, TryFromPrimitive};
+use strum_macros::EnumIter;
+
+#[cfg(feature = "python")]
+use pyo3::prelude::*;
+
+/// Registers for the ARM instruction set.
+#[derive(IntoPrimitive, TryFromPrimitive, Debug, Clone, Copy, EnumIter)]
+#[repr(i32)]
+pub enum Aarch64Regs {
+    R0 = 0,
```

Comment:
aarch64 regs are called x0-xN... And you can access the 16 bit part of them individually.